.github/workflows/express-build.yml

```diff
@@ -0,0 +1,110 @@
+# GitHub Action Workflow for building on Linux and Windows
+
+name: ExpressJS Express Build
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  
+  build:
+    name: Express Build
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: ["0.10", "0.12", "1.8", "2.5", "3.3", "4.9", "5.12", "6.17", "7.10", "8.17", "9.11", "10.19", "11.15", "12.16"]
+        os: [ubuntu-latest, windows-latest]
+        experimental: [false]
+        node-mirror: ["https://nodejs.org/dist/"]
+        include:
+          - node-version: 13
+            os: ubuntu-latest
+            node-mirror: "https://nodejs.org/download/nightly"
+            experimental: true
+          - node-version: 13
+            os: windows-latest
+            node-mirror: "https://nodejs.org/download/nightly"
+            experimental: true
+            
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Set up node using nvm
+      uses: dcodeIO/setup-node-nvm@v4.0.0
+      with:
+        node-version: ${{ matrix.node-version }}
+        node-mirror: ${{ matrix.node-mirror }}
+      
+    # Configure npm
+    # Skip updating shrinkwrap / lock
+    - run: npm config set shrinkwrap false
```

Comment:
I have said this in other issues, but I strongly reccomend we need to remove this and just commit an `.npmrc` which disabled the lock.  This is something we should do across all libraries in all three express orgs.

Reply:
I would probably say that to keep the scope from creeping, this actions conversion should just convert what is done today into actions, and changes that that kept separate, which we can do either before or after the actions conversion.

Reply:
I am good with that!  I just thought I would bring it up again since it had repercussions here.

Reply:
This is where I'm gonna have to plead a little ignorance on the technology. I'm happy to do this, I just need a little hand holding or explanation.